open-build-service/notqmail-rpmlintrc

```diff
@@ -0,0 +1,14 @@
+addFilter(".* unversioned-explicit-self-provides")
+addFilter(".* non-standard-gid")
+addFilter(".* is not allowed anymore in FHS .*")
```

Comment:
This reminds us to [do something about FHS sometime](https://github.com/notqmail/notqmail/issues/2#issuecomment-509028214)

Reply:
Yes. This is about FHS and also make notqmail get accepted by debian and others so that qmailq, qmaill, qmails, qmailr ids and qmail, nofiles gid gets accepted as standard system uid and gids.